pacsman/filesystem_dev_client.py

```diff
@@ -30,7 +30,7 @@
 from pydicom.valuerep import MultiValue
 
 from .dicom_interface import DicomInterface
-
+from .utils import process_and_write_png
 
 logger = logging.getLogger(__name__)
 
```

Comment:
Why is `dicom_datasets` a global, instead of a property of the `FilesystemDicomClient` instance?

Reply:
I think was so the datasets wouldn't have to be loaded twice, once for the local and once for the remote client (although right now it's not checking if the filepath has already been loaded, so it loads it twice anyway).

Reply:
What if we wanted multiple file system clients, each looking in different dicom directories?

BTW, Yujan thinks the `FilesystemClient` will be used in production scenarios.  E.g., Mobius has their own implementation of a file system client similar to ours.  Thus, at some point we will not want to store all of the datasets in memory (because there could be 1000s of them).  I don't think we need to worry about this for now, but I thought you may want to know for future reference.

Reply:
The source dir is also hard-coded at module level right now. 

Building the indexes for the future high-capacity version is definitely something to keep in mind.